management_api_app/api/routes/workspace_service_templates.py

```diff
@@ -28,6 +30,22 @@ async def get_workspace_service_templates(
     return ResourceTemplateInformationInList(templates=workspace_service_templates)
 
 
+@router.get("/workspace-service-templates/{template_name}", response_model=WorkspaceServiceTemplateInResponse, name=strings.API_GET_WORKSPACE_SERVICE_TEMPLATE_BY_NAME)
+async def get_current_workspace_template_by_name(
```

Comment:
name does not follow the route. It does not need to but it reads better if it is.

Reply:
fixed